CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ endif()
 
 find_package(Qt5 COMPONENTS Widgets REQUIRED)
 find_package(Qt5 COMPONENTS Network REQUIRED)
+find_package(Qt5LinguistTools REQUIRED)
```

Comment:
Can it be `find_package(Qt5 COMPONENTS LinguistTools REQUIRED)` ?

That will be more consistent with the rest of find package calls.